plugins/pulp_rpm/plugins/migrations/0001_rpm_add_export_distributor.py

```diff
@@ -41,7 +42,7 @@ def _migrate_rpm_repositories():
             iso_distributor = collection.find_one(
                 {'repo_id': repo_distributor['repo_id'], 'distributor_type_id': 'iso_distributor'})
             if iso_distributor is not None:
-                collection.remove(iso_distributor)
+                collection.remove(iso_distributor, safe=True)
```

Comment:
safe=True is deprecated. @dkliban I am failing to recall, didn't we build the necessary machinery to provide the correct write concern to the connection? If so, @asmacdo I think you should not use safe= here. Also, I wouldn't wait on hearing from @dkliban about this and just go ahead and drop the safe for now. When @dkliban responds we can open a new PR if needed (I believe it is not needed).